src/open_inwoner/conf/parts/maileditor.py

```diff
@@ -217,8 +217,8 @@ def _readfile(name):
                 "description": _("The description of the type of the case"),
             },
             {
-                "name": "case_link",
-                "description": _("The link to the case details."),
+                "name": "status_description",
+                "description": _("The description of the status of the case"),
```

Comment:
Note: `case_link` appeared twice in this list

Reply:
Woops, well done!